Format None, True, False and NotImplemented as literals (#118758)
python/cpython: Misc/NEWS.d/3.11.0b1.rst

.. nonce: l1p7CJ
.. section: Library

> For ``@dataclass``, add ``weakref_slot``. Default is ``False``. If true, and if

Reviewer: ```suggestion
For :func:`@dataclass <dataclasses.dataclass>`, add *weakref_slot*.
The new parameter defaults to ``False``. If true, and if
```
